redis/desc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<desc xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../_scripts/desc.xsd">
+   <antlr-version>^4.10</antlr-version>
+   <targets></targets>
```

Comment:
Please change to `<targets>Antlr4ng;CSharp;Cpp;Dart;Go;Java;JavaScript;PHP;Python3;TypeScript</targets>` so this grammar can be tested. Also, remove the `<grammar-files>` elements. They aren't needed because the tester "sees" the .g4s in the directory, and just "works" with them.